Puzzlepart/prosjektportalen-program
# Fixing program portfolio

Close to identical in pp main

### Your checklist for this pull request

- [x] Make sure you are requesting to **pull a feature/bugfix/issue branch** (right side).

### Description

Added styling to div #pp-program-portfolio.

### How to test

Install and build.
Note: There appears to be an issue with having a separate compiled .css file that is not main.css - pzlcloud has a program.css file, but seems not to be linked. May need to be linked before change will be viewed.

### Relevant issues (if applicable)

Closes #44
